src/current/v23.1/goldengate.md

```diff
@@ -0,0 +1,527 @@
+
+    {% include_cached copy-clipboard.html %}
+    ~~~
+    [CRDBSERVERLESS]
+    # The following driver will always point to your Oracle GoldenGate for PostgreSQL installation
+    Driver=/u01/ggs-pg/lib/GGpsql25.so
+    Description=DataDirect 7.1 PostgreSQL Wire Protocol
+    Database={hostname}.{database}
+    HostName={host}
+    PortNumber=26257
+    LogonID={your sql user}
+    Password={your sql user's pass}
+    EncryptionMethod=1 
+    ValidateServerCertificate=1 
+    TrustStore=/root/.postgresql/root.crt
+    ~~~
+        
+1. Log in to the database:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cd /u01/ggs-pg
+
+    ./ggsci
+
+    # To log into the local database
+    DBLOGIN SOURCEDB CRDBLOCAL
+
+    # To log into the serverless database and then enter your password
+    DBLOGIN SOURCEDB CRDBSERVERLESS
+    ~~~     
+
+## Set up Extract to capture data from a source database
+
+[Extract](https://docs.oracle.com/goldengate/c1230/gg-winux/GGCON/processes-and-terminology.html) is Oracle GoldenGate's data capture mechanism that is configured to run against the source database.
+
+{{site.data.alerts.callout_success}}
+Complete the steps in this section on a machine and in a directory where Oracle GoldenGate for Oracle is installed.
+{{site.data.alerts.end}}
+
+1. In the GGSCI terminal, create and open the `epos` parameter file for the Oracle source:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cd /u01/ggs # Otherwise known as $OGG_HOME
+    ./ggsci
+
+    edit param epos
+    view param epos
+    ~~~
+
+1. Edit the parameters that configure Extract to send data to the trail file at `./dirdat/ab` on the remote host:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~
+    EXTRACT epos
+    USERIDALIAS gg_source
+    RMTHOST {host-name}, MGRPORT {port}
+    RMTTRAIL ./dirdat/ab
+    TABLE OGGADM1.testtable;
+    ~~~
+
+1. In the GGSCI terminal, run the following and start the Extract service.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    add extract epos, tranlog, begin now
+    add rmttrail ./dirdat/ab, extract epos megabytes 10
+
+    # This is crucial in order to address the logmining not found error
+    register EXTRACT epos, DATABASE
+
+    # Starts service
+    start epos
+    ~~~
+
+1. Check the status of the Extract by creating a test table `OGGADM1.testtable` in Oracle and adding a row:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE TABLE OGGADM1.testtable (col1 number, col2 varchar2(20));
+    ALTER TABLE OGGADM1.testtable ADD PRIMARY KEY (col1);
+    INSERT INTO OGGADM1.testtable (col1, col2) VALUES (11, 'Example data');
+    COMMIT;
+    ~~~
+
+1. In GGSCI, in another terminal, check that Extract is working correctly:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
```

Comment:
I have a feeling that this snippet simply needs to be `stats EXTRACT EPOS` to produce the following output. See Step 9 in the Replicat section.

Currently, this looks like a repeat of steps 1 and 2 in this section. Maybe @ryanluu12345 can confirm.

Reply:
Yep, you can just use `stats EXTRACT EPOS` since that will show the output of how many CRUD operations happened. The `view param epos` is a good sanity check to make sure there is a process named epos running, but is entirely optional. You can probably include it as an optional sanity check step